libs/features/mas/commerce/test/checkout.test.js

```diff
@@ -88,15 +88,15 @@ describe('class "CheckoutLink"', () => {
         );
     });
 
-    it('renders link with ims country', async () => {
+    it('renders link with page country, not ims country', async () => {
         mockIms('CH');
         const service = await initMasCommerceService();
         const checkoutLink = mockCheckoutLink('abm');
         await service.imsCountryPromise;
         await delay(1);
         await checkoutLink.onceSettled();
         expect(checkoutLink.href).to.equal(
-            'https://commerce.adobe.com/store/email?items%5B0%5D%5Bid%5D=632B3ADD940A7FBB7864AA5AD19B8D28&cli=adobe_com&ctx=fp&co=CH&lang=en',
```

Comment:
Tests are fully reverted. So what I did is :
On initial render we use page country and there are no problems. 
When IMS country is set in `data-ims-country` we have re-render and then before WCS offers are loaded I need to override IMS country with page country `overrides.imsCountry = null`
After WCS offers are loaded (no new WCS call is triggered since the WCS url is identical as in the first render)
and before checkout link is generated I need to set the country back to IMS country
`options.country = this.dataset.imsCountry || options.country`

Is it ok now?